src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -57,20 +56,35 @@ public BtConnectorHelper() {
         this.myPeerName = peerName;
         this.lastAvailableList.clear();
         Stop();
-        mBTConnector = new BTConnector(context,this,this,conSettings,instanceEncryptionPWD);
+        mBTConnector = new BTConnector(context,this,this,conSettings);
         return mBTConnector.Start(this.myPeerIdentifier,this.myPeerName);
     }
 
+    public boolean isRunning(){
+        if(mBTConnector != null){
+            return true;
+        }else{
+            return false;
+        }
+    }
     public void Stop(){
```

Comment:
Are we sure that the calls to stop below can never throw? I'm asking because if there is any problem then we will have an uncaught exception that will leave us in an undefined state and necessitate crashing the app that is using us. That's fine if we are in some horrible unrecoverable mess but otherwise we should do our best to keep running.